setup.py

```diff
@@ -1,22 +1,18 @@
-# Copyrighted by The University of Texas at Dallas subject to the United States 
+# Copyrighted by The University of Texas at Dallas subject to the United States
 # government unlimited rights (DFAR 252.227-7013-5).
 
 import setuptools
```

Comment:
package not used

Reply:
Turns put python setup.py develop needs this import statement to work, even though the package isn't obviously used within setup.py